pkgs/development/embedded/arduino/arduino-core/chrootenv.nix

```
@@ -0,0 +1,34 @@
+{ lib, buildFHSUserEnv, arduino-core-unwrapped, withGui ? false, withTeensyduino ? false }:
+
+buildFHSUserEnv {
+  name = "${arduino-core-unwrapped.pname}-${arduino-core-unwrapped.version}";
+
+  targetPkgs =
+    pkgs: (with pkgs; [
+      ncurses
+      arduino-core-unwrapped
+      zlib
+      (python3.withPackages (p: with p; [
+        pyserial
+      ]))
+    ]);
+  multiPkgs = null;
+
+  extraInstallCommands = ''
+    ${lib.optionalString withGui ''
+      # desktop file
+      mkdir -p $out/share/applications
+      cp ${arduino-core-unwrapped.src}/build/linux/dist/desktop.template $out/share/applications/arduino.desktop
+      substituteInPlace $out/share/applications/arduino.desktop \
+        --replace '<BINARY_LOCATION>' "$out/bin/arduino" \
+        --replace '<ICON_NAME>' "$out/share/arduino/icons/128x128/apps/arduino.png"
+      # icon file
+      mkdir -p $out/share/arduino
+      cp -r ${arduino-core-unwrapped.src}/build/shared/icons $out/share/arduino
+    ''}
+  '';
+
+  passthru.runScript = "arduino";
```

I think should not have added passthru here
```suggestion
  runScript = "arduino";
```

---

still results in `arduino-core`